src/main/java/com/angelbroking/smartapi/smartstream/ticker/SmartStreamTicker.java

```diff
@@ -30,7 +30,10 @@
 @Slf4j
 public class SmartStreamTicker {
 
-	private static final int PING_INTERVAL = 10000; // 10 seconds
+	private static int PING_INTERVAL = 10000; // 10 seconds

```

Comment:
make variable name logically which can show seconds, ms etc.

Reply:
Variables names are now updated according to its usage. Please check.